### PR TITLE
Add Ukrainian translation

### PR DESCRIPTION
**Pull request by McGiverGim**
Thanks to the translators, specially to Oleg Shevtsov for the hard part.

image

### CODE REVIEW

**Comment by McGiverGim** [comment]
Changed the text as requested. Thanks!

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**
